app/src/pages/prompt/PromptInvocationParameters.tsx

```diff
@@ -0,0 +1,72 @@
+import React, { useMemo } from "react";
+import { isObject } from "lodash";
+import { css } from "@emotion/react";
+
+import { Flex, Text, View } from "@arizeai/components";
+
+function PromptInvocationParameterItem({
+  keyName,
+  value,
+}: {
+  keyName: string;
+  value: unknown;
+}) {
+  return (
+    <View padding="size-100">
+      <Flex direction="row" justifyContent="space-between">
+        <Text weight="heavy">{keyName}</Text>
+        <Text>{String(value)}</Text>
+      </Flex>
+    </View>
+  );
+}
+
+const promptInvocationParametersCSS = css`
+  // every other row should have a background color
+  & > *:nth-child(even) {
+    background-color: var(--ac-global-background-color-light);
+    border-radius: 4px;
+  }
+`;
+
+type PromptInvocationParametersProps = {
+  invocationParameters: Record<string, unknown>;
+};
```

Comment:
It's hard to decide when to pull in a fragment + fragment key type vs accepting general purpose types as props. This component will be getting its data from relay either way, but as fragments from two separate queries

Reply:
nvm figured this out. I can always use a fragment if the data is coming from relay.

As long as I pass query results down from some query that spreads the fragment name I am good to go